src/services/armService.ts

```diff
@@ -97,7 +97,7 @@ export class ArmService extends BaseService {
     Object.keys(deployment.parameters).forEach((key) => {
       const parameterValue = deployment.parameters[key];
       if (parameterValue) {
-        deploymentParameters[key] = { value: deployment.parameters[key] };
+        deploymentParameters[key] = (typeof parameterValue === "string") ? { value: parameterValue } : { value: parameterValue.value }
```

Comment:
@wbreza When I get the ARM template from the previous deployment, the parameter values look like:
```
{
   type: "String",
   value: "value1"
}
```
When I just used that object rather than extracting the value, it fails. I guess it doesn't like the `type` attribute for some reason. Do you have any cleaner ways for how to do this?

Reply:
The `deployTemplate(...)` is called it is called with deployment params of simple key/value pairs and then converted into the final format ARM templates expect by default.  What I imagine is happening is you are passing the params as already final constructed ARM params from the pre-existing deployment.